src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocationsTests.cs

```diff
@@ -2002,63 +2000,94 @@ async void M(CancellationToken ct)
             return VerifyCS.VerifyCodeFixAsync(originalCode, fixedCode);
         }
 
-        [Theory]
-        [InlineData("CancellationToken c", "", "", "")]
-        [InlineData("", "CancellationToken c", "", "")]
-        [InlineData("", "", "<CancellationToken>", "c")]
-        public Task CS_Diagnostic_MultiNesting(string topMethodParam, string localMethodParam, string extensionTypeParam, string extensionArg)
+        [Fact]
```

Comment:
I split the "MultiNesting" methods into two separate test methods:
- One that verifies the ct being passed from the parent local method.
- One that verifies the ct being passed from the top method, skipping the local method.

The case for the lambda is no longer offering a diagnostic.